fastlane/Fastfile

```diff
@@ -613,54 +668,6 @@ platform :ios do
     )
   end
 
-  #####################################################################################
-  # register_new_device
-  # -----------------------------------------------------------------------------------
-  # This lane helps a developer register a new device in the App Store Portal
-  # -----------------------------------------------------------------------------------
-  # Usage:
-  # bundle exec fastlane register_new_device
-  #
-  # Example:
-  # bundle exec fastlane register_new_device
-  #####################################################################################
```

Comment:
Replaced by the [MC tool](https://mc.a8c.com/mobile/app-store-connect/devices/).